src/main/java/Task.java

```diff
@@ -18,4 +21,12 @@ public boolean getDone(){
     public String getDescription(){
         return description;
     }
+    public String toString(){
+        if(isDone == true) {
+            reply = "\u2713";
+        } else {
+            reply = "\u2718";
+        }
+        return ("[" + reply + "] " + description); //return tick or X symbols
```

Comment:
You can call getStatusIcon() to get the status instead of checking isDone again.